<div class="container">

<table style="width: 100%;"><tr>
<td>throw.Exception</td>
<td style="text-align: right;">R Documentation</td>
</tr></table>
<h2>Throws an Exception that can be caught</h2>

<h3>Description</h3>

<p>Throws an Exception that can be caught by <code>tryCatch()</code>.
</p>


<h3>Usage</h3>

<pre><code class="language-R">## S3 method for class 'Exception'
throw(this, ...)
</code></pre>


<h3>Arguments</h3>

<table><tr style="vertical-align: top;">
<td><code>...</code></td>
<td>
<p>Not used.</p>
</td>
</tr></table>
<h3>Value</h3>

<p>Returns nothing.
</p>


<h3>Author(s)</h3>

<p>Henrik Bengtsson</p>


<h3>See Also</h3>

<p>For more information see <code>Exception</code>.
See also <code>tryCatch()</code>.
This method overrides (and is fully backward compatible with) the one
defined in the <span class="pkg">R.methodsS3</span> package.
</p>


<h3>Examples</h3>

<pre><code class="language-R">## Not run: For a complete example see help(Exception).</code></pre>


</div>